tasks/test-project/test-project

```diff
@@ -101,6 +107,22 @@ const createProject = async () => {
   )
 }
 
+const copyProject = async () => {
+  if (clean) {
+    rimraf.sync(OUTPUT_PROJECT_PATH)
+  }
+
+  const FIXTURE_TESTPROJ_PATH = path.join(
+    RW_FRAMEWORKPATH,
+    '__fixtures__/test-project'
+  )
+
+  await fse.copy(FIXTURE_TESTPROJ_PATH, OUTPUT_PROJECT_PATH)
+
+  // Make sure no lockfiles are accidentally copied
+  fse.remove(path.join(OUTPUT_PROJECT_PATH, 'yarn.lock'))
+}
+
 const globalTasks = () =>
   new Listr(
     [
```

Comment:
Looks like you can remove L129-138 as there's no longer a separate package.json for this script (or any tasks). This will just be a redundant `yarn install` after the step `name: Install framework dependencies` in `smoke-test.yaml`, correct?

Reply:
Yes that's right! I was wondering whether to remove... but its kinda useful when we switch branches, etc.

Reply:
I'd say scratch it for sure